### PR TITLE
Mimo poles

### PR DESCRIPTION
**Pull request by albheim**
Implementation of MIMO poles according to Glad and Ljung book. Implementation can definitely be improved, but seems to be working at least.

### CODE REVIEW

**Comment by mfalt** [comment]
Do you know what failed and why? It would be nice if we could pull this.

File: src/analysis.jl [hunk 1]
```diff
@@ -2,10 +2,89 @@
...
+        end
+    end
+    return minors
+end
+
+"""`det(sys)`
+
+Compute the determinant of the Matrix `sys` of SisoTf systems, returns a SisoTf system."""
+# TODO: improve this implementation, should be more efficient ones
+function det(sys::Matrix{SisoZpk{T, TR}}) where {T<:Number, TR}
```
Comment (mfalt) [inline src/analysis.jl]: Should be `Matrix{S} ... where {T,TR,S<:SisoZpk}` since S is used later.

File: test/test_analysis.jl [hunk 2]
```diff
@@ -177,6 +179,15 @@ z, p, k = zpkdata(G)
      "|  -2.000e+00   |  1.000e+00    |  2.000e+00    |  5.000e-01    |\n"*
      "|  3.000e+00    |  -1.000e+00   |  3.000e+00    |  -3.333e-01   |\n")
 
+s = tf("s")
```
Comment (mfalt) [inline test/test_analysis.jl]: This line can be removed

**Comment by mfalt** [comment]
Should be ready to pull if no one has any comments.

**Comment by baggepinnen** [comment]
Ready to pull?

**Pull request merged**